server/src/main/java/com/flipkart/varadhi/RestVerticle.java

```diff
@@ -36,11 +44,32 @@ public void start(Promise<Void> startPromise) {
         log.info("HttpServer Starting.");
         Router router = Router.router(vertx);
 
+        // grpc
+        GrpcServer grpcServer = GrpcServer.server(vertx);
+
+        grpcServer.callHandler(MessageProducerGrpc.getProduceMethod(), request -> {
+            request.response().status(GrpcStatus.OK).statusMessage("OK")
+                    .end(SingleMessageResponse.newBuilder().setOffset("0001").build());
+        });
+
+        router.route()
+                .consumes("application/grpc")
+                .produces("application/grpc")
+                .handler(req -> grpcServer.handle(req.request()));
+
+        router.route()
+                .consumes("application/json")
+                .produces("application/json")
+                .path("/topics/produce")
+                .method(HttpMethod.POST)
+                .handler(rc -> Extensions.RoutingContextExtension.endRequestWithResponse(rc, 200, "0001"));
+
```

Comment:
Planning to keep it or move it appropriately to ?

Reply:
ya. its a TODO, will fix it in this PR.